Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Startup.cs

```diff
@@ -32,25 +39,28 @@ public void ConfigureServices(IServiceCollection services)
 
             var connection = @"Server=dev.retrotest.co.za;Database=ereader;User Id=group4;Password=3bHNuE8&rvG+99U2;";
             services.AddDbContext<MyDbContext>
-                (options => options.UseSqlServer(connection));
+                (options => options.UseSqlServer(connection)); //other option: UseSqlServer(Configuration.GetSection(connection))
             services.AddSingleton(Configuration);
         }
 
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
         public void Configure(IApplicationBuilder app, IHostingEnvironment env)
         {
-            // Enable middleware to serve generated Swagger as a JSON endpoint.
-            app.UseSwagger();
+            app.UseCors("CorsPolicy");
+            // UpdateDatabase(app);
```

Comment:
I can't see a problem apart from this Ghost Code

Reply:
Thank you Fred. I will remove the ghost code. Were primarily just trying to get the cors problem solved.